// register controller
const registerUser = async (req, res) => {
  try {

  } catch (error) {
    console.log(e)
    res.status(500).json({
      success: false,
      message: "Something went wrong"
    })
  }
}

//login controller
const loginUser=async (req,res) => {
  try {
    
  } catch (error) {
    
  }
}